JavaScript/Github_Fetch_Issue_Details/.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+node_modules
```

Comment:
A separate `.gitignore` is not recommended. We have a directory-wide ignore file already. Please delete this file, and make any changes in the home gitignore. (Please explain the reason as well for the same)

Reply:
Removing that .gitignore, Directory wide .gitignore already contains the variables contained in mine, So there is no need for any change. Thanks for mentioning this out.